### ISSUE TITLE
Automatic doc of Scylla scripts

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#8428]
Many (all?)of scylla scripts are well documented using standard -h options
Most of this scripts are lacking from Scylla docs, or miss some of the options the scripts have.

It would be good to automatically use the -h options from the script to generate a static doc page per release, which will be easy to find.

**Comment by tzach** [comment]
Related issues:
https://github.com/scylladb/scylla-doc-issues/issues/718
https://github.com/scylladb/scylla-doc-issues/issues/720

**Comment by tzach** [comment]
https://github.com/scylladb/scylla-docs/issues/3076

**Comment by tzach** [comment]
https://github.com/scylladb/scylla-doc-issues/issues/719

**Comment by slivne** [comment]
have we moved the docs of scylla to be in tree ?

**Comment by tzach** [comment]
Not all, but some are already in tree, for example Alternator
http://scylla.docs.scylladb.com/master/alternator/alternator.html

As such, we can do the same for scripts, config etc

**Comment by nyh** [comment]
I agree that "scylla --help", for example, has a lot of helpful information and better than nothing. But when it becomes part of our documentation we will probably need to start an effort to improve it. @eliransin already noted that the **defaults** of many of the options are only sporadically documented, and moreover a lot of these options would benefits from links to longer documents explaining their purpose, and also for some text talking about a group of options (e.g., all the options controlling the various ports and IP addresses have a longer explanation in protocols.md). I agree that as soon as the output of "scylla --help" becomes a documentation page, we'll have more motivation to improve it.
